pkg/liqo-controller-manager/webhooks/firewallconfiguration/table.go

```diff
@@ -41,7 +42,8 @@ func checkUniqueChainName(chains []firewallapi.Chain) error {
 
 // checkImmutableTableName checks if the table name is immutable.
 func checkImmutableTableName(fwcfg, oldFwcfg *networkingv1alpha1.FirewallConfiguration) error {
-	if oldFwcfg.Spec.Table.Name != fwcfg.Spec.Table.Name {
+	if fwcfg.Spec.Table.Name != nil && oldFwcfg.Spec.Table.Name != nil &&
```

Comment:
do we accept nil names? if not, you should check it in a separate if and return an error